src/components/clients/Client.js

```diff
@@ -12,10 +12,38 @@ class Client extends React.Component {
   }
 
   render() {
+    let authRedirectUrls = [];
+    if (this.props.authRedirectUrls) {
+      for (let i = 0; i < this.props.authRedirectUrls.length; i++) {
+        authRedirectUrls.push(
+          <p key={i}>
+            <a href={this.props.authRedirectUrls[i]}>
+              <small>{this.props.authRedirectUrls[i]}</small>
```

Comment:
wondering why you use `<small>` instead of using some easy CSS to target this ?

For example you could add a class to the 2 `<td>`s (eg: `table-redirect-url`).

(also below)